packages/dev/inspector-v2/src/components/properties/animation/animationsProperties.tsx

```diff
@@ -160,14 +167,49 @@ export const AnimationsProperties: FunctionComponent<{ scene: Scene; entity: Par
                                     }
                                 }}
                             />
+                            {ranges.length > 0 || animations.length > 0 ? (
+                                <>
+                                    <SwitchPropertyLine
+                                        label="Enable override"
+                                        value={animationPropertiesOverride != null}
+                                        onChange={(value) => {
+                                            if (value) {
+                                                mainAnimatableWithPropertiesOverride.animationPropertiesOverride = new AnimationPropertiesOverride();
+                                                mainAnimatableWithPropertiesOverride.animationPropertiesOverride.blendingSpeed = 0.05;
+                                            } else {
+                                                mainAnimatableWithPropertiesOverride.animationPropertiesOverride = undefined;
+                                            }
+                                        }}
+                                    />
+                                    {animationPropertiesOverride != null && (

```

Comment:
If you want this to be a little fancier, you can use a `<Collapse>` to make this content animation in/out. You can find other examples in the code. Up to you, it's a nice extra bit of polish.